src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/rc_pending_resumption_handler.cc

```diff
@@ -132,6 +131,11 @@ void RCPendingResumptionHandler::HandleSuccessfulResponse(
     }
     freezed_resumptions_.erase(it_queue_freezed);
   }
+  std::remove_if(subscriptions_.begin(),
+                 subscriptions_.end(),
+                 [&module_uid](const ModuleUid& module_to_remove) {
+                   return module_to_remove == module_uid;
+                 });
```

Comment:
I'm worried that after this successful request, we may get another subscription to the same module, and after these changes we will repeat the HMI request for a successful subscription again.

Reply:
After a successful response, we should put the module into the apps subscriptions ( in extension)
Nex resumption need to check if any other app is subscribed for the module before passing this list to rc_pending_resumption_handler